### DISCUSSION TITLE
CouchDB v2.3.1 can't install on centOS

### DISCUSSION THREAD

**Discussion by itsukix3** [apache/couchdb#3670] [Q&A] [answered]
I've run the installation on a docker container, and the command I run was below:

```
RUN echo -e "[bintray--apache-couchdb-rpm]\nname=bintray--apache-couchdb-rpm\nbaseurl=http://apache.bintray.com/couchdb-rpm/el\$releasever/\$basearch/\ngpgcheck=0\nrepo_gpgcheck=0\nenabled=1" >> /etc/yum.repos.d/bintray-apache-couchdb.repo
RUN yum -y install couchdb-2.3.1-1.el7.x86_64
```

But I got an error and can't install couchdb.
>http://apache.bintray.com/couchdb-rpm/el7/x86_64/repodata/repomd.xml: [Errno 14] HTTP Error 403 - Forbidden
>Trying other mirror.

Dose anyone know about it?

**Comment by wohali** [comment, marked as answer]
We had to move our repositories because bintray shut down.

See https://docs.couchdb.org/en/latest/install/unix.html for the latest instructions.

**Reply by itsukix3** [reply]
Thanks a lot.
Normally installed now.
